client/lib/signup/step-actions/index.js

```diff
@@ -557,28 +563,26 @@ export function addDomainToCart(
 
 	const newCartItems = [ domainItem, googleAppsCartItem ].filter( ( item ) => item );
 
-	processItemCart(
+	processItemCart( {
 		providedDependencies,
 		newCartItems,
 		callback,
 		reduxStore,
 		slug,
-		null,
-		null,
-		null
-	);
+	} );
 }
 
-function processItemCart(
+function processItemCart( {
 	providedDependencies,
 	newCartItems,
 	callback,
 	reduxStore,
 	siteSlug,
 	isFreeThemePreselected,
 	themeSlugWithRepo,
-	lastKnownFlow
-) {
+	themeStyleVariation,
```

Comment:
[nit] I think the first five are required, so we can leave it as it is and change the rest to object. But I'm fine with the current way 🙂